tests/feature/gherkin_terminal_reporter.feature

```diff
@@ -43,5 +43,10 @@ Feature: Gherkin terminal reporter
 
   Scenario: Should step parameters be replaced by their values
     Given there is gherkin scenario outline implemented
-    When tests are run with step expanded mode
+    When tests are run with step expanded option
```

Comment:
It would make it clearer to have a step like "when tests are run with --gherkin-terminal-reporter-expanded". What do you think?